RIOT-OS/RIOT
# pkg/umorse: Add Kconfig support

### Contribution description

This models the `umorse` package in Kconfig.

### Testing procedure

```
make clean -C tests/pkg_umorse/ 
TEST_KCONFIG=1 make menuconfig -C tests/pkg_umorse/ 
TEST_KCONFIG=1 make all test -C tests/pkg_umorse/
```

And check the CI

### Issues/PRs references
Depends on #15984
